compiler_opt/rl/corpus.py

```diff
@@ -30,9 +32,47 @@ class ModuleSpec:
   """
   name: str
   exec_cmd: Tuple[str, ...] = ()
+  size: int = 0
 
 
-def build_modulespecs_from_datapath(
+class Corpus:
+  """Represents a corpus. Comes along with some utility functions."""
+
+  def __init__(self,
```

Comment:
only support one way of initialization?

Reply:
do you mean adding a way for composing corpora together?

Reply:
i mean currently the __init__ is a bit confusing cause you are allowing both passing in a modulespec list or passing in data_path, additional_flag, delete_flag to construct the modulespec. this interface is confusing to users. you can either: 1) only support init from one way; or 2) have separate init for these two ways